test/integration/fixtures/blocks/core__image__deprecated-align-wrapper.html

```diff
@@ -1,3 +1,3 @@
-<!-- wp:image {"align":"left","id":13,"width":358,"height":164,"sizeSlug":"large","linkDestination":"none","style":{"border":{"radius":"10px"}}} -->
-<div class="wp-block-image" style="border-radius:10px"><figure class="alignleft size-large is-resized"><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAACklEQVR4nGMAAQAABQABDQottAAAAABJRU5ErkJggg==" alt="" class="wp-image-13" width="358" height="164"/></figure></div>
+<!-- wp:image {"align":"left","id":13,"width":358,"height":164,"sizeSlug":"large","linkDestination":"none"} -->
```

Comment:
I've removed the border-radius style from this fixture as it wasn't related to the align wrapper deprecation. It's now covered by a dedicated deprecation required by this PR. See: [test/integration/fixtures/blocks/core__image__deprecated-border-radius-5.html](https://github.com/WordPress/gutenberg/pull/31366/files#diff-92e518972d78f9d6d1e174ff6a2331d7eb200e62aafedbd7ba243ad0248339b0)